sample_metadata/parser/cloudhelper.py

```diff
@@ -89,7 +92,7 @@ def list_directory(self, directory_name) -> list[str]:
     async def file_contents(self, filename) -> str | None:
         """Get contents of file (decoded as utf8)"""
         path = self.file_path(filename)
-        with AnyPath(path).open() as f:
+        with AnyPath(path).open(encoding='utf-8-sig') as f:
```

Comment:
Interesting -- why `utf-8-sig` here instead of `utf-8`? According to https://stackoverflow.com/questions/2223882/whats-the-difference-between-utf-8-and-utf-8-without-bom, a BOM isn't generally recommended.

Reply:
Specifically the OHMR4 template has a BOM in it - and I don't think it breaks the other way. I don't have a good way to detect it when reading and it causes issues with forming the key map.